Octokit/Models/Request/SearchCodeRequest.cs

```diff
@@ -18,6 +18,12 @@ public class SearchCodeRequest : BaseSearchRequest
     {
         public SearchCodeRequest(string term) : base(term) { }
 
+        public SearchCodeRequest(string term, string repo)
```

Comment:
Yes, this should include the `owner` property.

For example, https://api.github.com/search/code?q=addClass+repo:jquery returns a validation error. But https://api.github.com/search/code?q=addClass+repo:jquery/jquery returns an array of results.